### PR TITLE
Adding support for the minimum edge distance

### PR DESCRIPTION
**Pull request by mindmind**
Because app crashes in some cases when image was scaled to zero

### CODE REVIEW

**Comment by jywarren** [comment]
Noting that we'd have to re-open this against `main` branch because we're moving to that. Thanks!

**Comment by jywarren** [comment]
I think with a little more commenting we'd be happy to merge this!

**Comment by mindmind** [comment]
@jywarren That was a long time ago 😰
Should I write comments right here? or in code?

**Comment by jywarren** [comment]
haha thanks! Yes please! In the code would be great, just to make it clear
what the new section is for. Thanks!!!

**Comment by mindmind** [comment]
@jywarren done. its ok?

**Comment by jywarren** [comment]
OH this is super. Thank you! Actually, we will need the change to be made in the /src/ folder, not in the /dist/ folder, i'm sorry I didn't catch this. Would you mind moving the change there, then running `grunt concat:dist`? We really appreciate it! And can help out if you'd like. 👍

**Comment by mindmind** [comment]
@jywarren done!

File: src/edit/RotateHandle.js [hunk 1]
```diff
@@ -19,6 +19,21 @@ L.RotateHandle = L.EditHandle.extend({
 		overlay.editing._rotateBy(angle);
 		overlay.editing._scaleBy(scale);
 
+		/* 
+		  checks whether the "edgeMinWidth" property is set and tracks the minimum edge length;
+		  this enables preventing scaling to zero, but we might also add an overall scale limit
```
Comment (jywarren) [inline src/edit/RotateHandle.js]: I just added one more line of context here for the future... then this is good to merge! Thanks!

Comment (mindmind) [reply src/edit/RotateHandle.js]: Yes, i think we need to insert hard limit on minimum scale to prevent errors. I can do it

**Comment by jywarren** [comment]
That'd be awesome, thanks!

**Pull request merged**